candi/version_map.yml

```diff
@@ -138,9 +138,9 @@ k8s:
     patch: 16
     cniVersion: 0.8.7
     bashible: &bashible_k8s_ge_1_19
-      <<: *bashible
+      !!merge <<: *bashible
```

Comment:
What is !!merge?

Reply:
Tag !!merge adds automatically by yq parser for backward compatibility with yaml spec v1.1.
More info - https://mikefarah.gitbook.io/yq/usage/tips-and-tricks#yq-adds-a-merge-tag-automatically